.github/workflows/continuous-integration.yml

```diff
@@ -25,13 +29,72 @@ jobs:
       with:
         dotnet-version: 3.1.302
 
-    - name: Install Dependencies
+    - name: Install dependencies
       run: dotnet restore
       env:
         DOTNET_NOLOGO: 1
 
     - name: Build
-      run: dotnet build --configuration Release --no-restore
+      run: dotnet build --configuration Release --no-restore -p:UseAppHost=true # Force generation of executable on macOS.
 
-    - name: Unit Test
+    - name: Unit test
       run: dotnet test --no-restore
+
+    - name: Setup platform (Linux)
+      if: runner.os == 'Linux'
+      run: echo "::set-env name=BUILD_PLATFORM::${{ runner.os }}"
+
+    - name: Setup platform (Mac)
+      if: runner.os == 'macOS'
+      run: echo '::set-env name=BUILD_PLATFORM::Mac'
+
+    - name: Setup platform (Windows)
+      if: runner.os == 'Windows'
+      run: |
+        echo "::set-env name=BUILD_PLATFORM::${{ runner.os }}"
+        echo '::set-env name=BUILD_FILE_EXT::.exe'
+
+    - name: Setup Git installer
+      shell: bash
+      run: |
+        GIT_VERSION=$(grep '<GitPackageVersion>' Directory.Build.props | grep -Eo '[0-9.]+(-\w+)*')
+        cd ../out
+        dotnet new classlib -n Scalar.GitInstaller
+        cd Scalar.GitInstaller
+        cp ../../scalar/nuget.config .
+        dotnet add Scalar.GitInstaller.csproj package "GitFor${BUILD_PLATFORM}.GVFS.Installer" --package-directory . --version "$GIT_VERSION"
```

Comment:
I think the difference is how you use `${ }` here. Interesting.

Reply:
Could be ... I was able to use a plain `$BUILD_FILE_EXT` below, though, so I don't know.